src/vcontrol.rs

```diff
@@ -32,14 +32,18 @@ impl VControl {
     };
 
     let mut buf = [0; 2];
-    protocol.get(&mut optolink, 0x00f0, &mut buf)?;
-    let f0 = u16::from_be_bytes(buf);
+    let f0 = match protocol.get(&mut optolink, 0x00f0, &mut buf) {
+      Ok(()) => u16::from_be_bytes(buf),
+      Err(e) => 0u16,
```

Comment:
What error do you expect to occur here? For my device at least, this just returns `[0xff, 0xff]`, which is covered by setting the `f0_till` to `0xffff` if it is undefined.

Reply:
For my device the application panicked with:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Custom { kind: InvalidData, error: "unexpected response length" })', src/bin/vcontrol.rs:50:47
```

The communication is in that case
```
write [0, 1, 0, 240, 2]
read [3, 1, 0, 240, 1, 1]
```

So it would be better to raise an `Error::UnsupportedCommand` here within the protocol. The `Error::UnsupportedCommand` should be treated as optional and the DeviceIdentF0 should be set to 0u16.